packages/vuetify/src/components/VDataTable/VDataTableHeaderDesktop.ts

```diff
@@ -16,76 +16,95 @@ export default mixins(header).extend({
         },
       }, ['group'])
     },
-    // eslint-disable-next-line max-statements
+    getAria (beingSorted: boolean, isDesc: boolean) {
+      let ariaSort = 'none'
+      let ariaLabel = [
+        this.$vuetify.lang.t('$vuetify.dataTable.ariaLabel.sortNone'),
```

Comment:
I would assign `$vuetify.dataTable.ariaLabel` to a string and use it in all of the lines below.